bedrock/firefox/templates/firefox/family/includes/modules/private-mode.html

```diff
@@ -39,8 +39,8 @@ <h4 class="c-title-uppercase">Things to talk about:</h4>
 
       <div class="c-blurb">
         <h4 class="c-blurb-title">Something to think about:</h4>
-        <p>Installing an internet monitoring service on younger kids’ devices could
-          help give you more peace of mind.</p>
+        <p>There are plenty of parental control/internet monitoring apps available, but is installing one really the right move?
+          <a rel="external" href="https://blog.mozilla.org/en/internet-culture/deep-dives/parental-controls-internet-safety-for-kids/">There’s actually a lot to consider.</a></p>
```

Comment:
Should we add some UTM params to this link?

Reply:
Couldn't hurt. I used that `{{ utm_params }}` thing.